test/unit/serverless/fixtures.js

```diff
@@ -338,6 +338,60 @@ const lambaV1InvocationEvent = {
   }
 }
 
+// Event sent by API Gateway to an authorizing Lambda function. This should not be classified as a web event.
+// https://docs.aws.amazon.com/apigateway/latest/developerguide/api-gateway-lambda-authorizer-input.html
+// Connection, Host, and Upgrade headers removed due to prettier and eslint complaints: prettier requires
```

Comment:
```suggestion
// Connection, Host, and Upgrade headers removed due to eslint complaints: it requires
```

not sure if you don't have main against this on your local branch, but prettier has been removed. it relies on neostandard which uses @stylistic for it's code formatting